lib/Core/Executor.cpp

```diff
@@ -1533,7 +1544,11 @@ void Executor::executeInstruction(ExecutionState &state, KInstruction *ki) {
         ref<Expr> value = evalConstant(si->getCaseValue(i));
 #endif
         ref<Expr> match = EqExpr::create(cond, value);
-        isDefault = AndExpr::create(isDefault, Expr::createIsZero(match));
+
+        // default value cannot contain this target
+        defaultValue = AndExpr::create(defaultValue, Expr::createIsZero(match));
```

Comment:
@MartinNowack This comment isn't helpful (at least to me). It should really say something like

```
Add to accumulating ``AndExpr``.
```

and have a comment on the declaration of ``isDefault`` that says

```
After iterating over all switch targets if ``isDefault`` can be evaluated to true then it possible for control flow to fall through to the default case
```